src/Types/BakerInfo.cs

```diff
@@ -0,0 +1,31 @@
+namespace Concordium.Sdk.Types;
+
+/// <summary>
+/// Information about a baker.
+/// </summary>
+/// <param name="BakerId">
+/// Identity of the baker. This is actually the account index of
```

Comment:
Nit: "actually" sounds unnecessarily surprised to me. Would just write something like "Account index of the account associated with the baker." It being the ID of the baker is implied in the name.